onnxruntime/python/tools/profile_explorer/profile_explorer.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+
+import argparse
+import json
+import subprocess as sp
+import pandas as pd
+
+
+def _demangle(name, demangler='cu++filt'):
+    try:
+        with sp.Popen([demangler, name], stdin=sp.PIPE, stdout=sp.PIPE) as proc:
+            out, _ = proc.communicate()
+            return out.decode('utf-8').strip()
+    except:
+        return name
+
+
+def _get_args():
+    parser = argparse.ArgumentParser(description='onnxruntime bench tool')
+    parser.add_argument('input', type=str, help='Trace input file, formatted as JSON')
+    parser.add_argument('--demangler', required=False, type=str, default='cu++filt', help='The command to use to demangle C++ identifiers')
```

Comment:
How about default `c++filt`? In case we will be using it for ROCm profiling.

Reply:
Sounds good, thanks!